src/map-layer.js

```diff
@@ -0,0 +1,68 @@
+export default {
```

Comment:
Move this into `/components` and make it a component.

Reply:
Why is this a separate component? Can't we just import this from '[vue2mapbox-gl](https://github.com/openearth/vue2mapbox-gl/tree/master/src/components)'?

Reply:
Hmm ... In the deltares-components we have only a MapboxWmsLayer but in this case I want to load a geojson layer instead of wms... Is there an update on vue2mapbox-gl that I haven't seen?